fvm/crypto/crypto.go

```diff
@@ -150,6 +150,22 @@ func CryptoToRuntimeHashingAlgorithm(h hash.HashingAlgorithm) runtime.HashAlgori
 	}
 }
 
+// ValidatePublicKey returns true if public key is valid
+func ValidatePublicKey(signAlgo runtime.SignatureAlgorithm, pk []byte) (valid bool, err error) {
+	sigAlgo := RuntimeToCryptoSigningAlgorithm(signAlgo)
+
+	_, err = crypto.DecodePublicKey(sigAlgo, pk)
+
+	if err != nil {
+		// TODO: `crypto.IsInvalidInputsError(err)` doesn't work here!
```

Comment:
What was the issue with using `crypto.IsInvalidInputsError(err)` ?

Reply:
It was returning false when the condition I use now returns true.

Reply:
fixed [here](https://github.com/onflow/flow-go/pull/788/commits/80e5fc3c10bf744bd0a569705601da5264e21d2f)